src/main.js

```diff
@@ -98,7 +98,7 @@ function toggleAllUsers(status = false) {
     if (!status) {
         userIdsToUnfollow = [];
     } else {
-        userIdsToUnfollow = nonFollowersList.map(user => user.id);
+        userIdsToUnfollow = nonFollowersList.map(user => parseInt(user.id, 10));
```

Comment:
Seems like the type of mistake that could have been avoided using typescript 😆 
Nicely done!

Reply:
Yep.. If we are all comfortable with TS I can migrate it.

Reply:
Yeah I definitely prefer it

Reply:
> Yeah I definitely prefer it

In the afternoon i will open a PR with the migration and some cleanups.